extensions/amp-story/0.1/progress-bar.js

```diff
@@ -110,12 +110,14 @@ export class ProgressBar {
     this.assertValidPageIndex_(pageIndex);
     for (let i = 0; i < this.pageCount_; i++) {
       if (i < pageIndex) {
+        // Fill progress bar for pages that precede the active page
         this.updateProgress(i, 1.0);
-      } else if (i > pageIndex) {
-        this.updateProgress(i, 0.0);
       } else {
-        // The active page manages its own progress by firing PAGE_PROGRESS
+        // The active page manages its own progress:
+        // if 'auto-advance-after' attribute exists it fires PAGE_PROGRESS
         // events to amp-story.
+        // If it doesn't then it just fills its own progress bar.
+        // All subsequent bars are un-filled.
         this.updateProgress(i, 0.0);
       }
     }
```

Comment:
The `else if` and the `else` both had the same output.

Can't fill the segment for the active page here since it gets overwritten by `TimeBasedAdvancement`